app/code/community/Reverb/ReverbSync/etc/config.xml

```diff
@@ -333,6 +333,12 @@
         <order_sync_reverb_source_url>/api/my/orders/selling/all?created_start_date=%s</order_sync_reverb_source_url>
         <!-- Default orders to be synced to the Admin store -->
         <store_to_sync_order_to>0</store_to_sync_order_to>
+        <!-- The set of order statuses which should be regarded as constituting NOT being a paid order -->
+        <non_paid_order_statuses>
```

Comment:
The assumption here is that people who are using this will be using Magento to determine when things get shipped, so in the case where something is already marked shipped in Reverb, they don't want to ship it again. In that case, we should only create orders that need to be shipped, which means things that are in the paid state only since that is the only state that says "payment is cleared and it has not been shipped yet".